.gitignore

```
@@ -1,3 +1,3 @@
 # Hier zu ignorierende Dateiendungen / Dateinahmen / Verzeichnisse auflisten!
 # automatisch generiert von MAC
-.DS_Store
+.DS_Store
```

```suggestion
.DS_Store

```
Leerzeile am Dateiende